### ISSUE TITLE
Bytecoin is a scam

### ISSUE THREAD

**Issue by Ben19904** [amjuarez/bytecoin#116]
I bought bytecoin two weeks ago. I bought it from a website called changer.com, and the transaction clearly completed (according to http://chainradar.com). I instructed it to be sent to my bytecoin wallet 1.1.1

at first I wasn't worried that I couldn't see my balance, because I knew I had to wait until wallet had synchronised. It took 7 days for this to happen, and when it did I noticed that my balance was still 0.00

after this I then realised the bytecoin team on twitter mentioned that they'd released a new wallet and reference client that fixed issues in regards to synchronization / blockchain corruption issues, so I thought maybe this had affected me.

i deleted the blockchain on my PC, downloaded the one via bytecoin.org, and then imported that into the new version of bytecoin wallet (1.1.4), and of course copied and pasted my wallet backup. into the relevant folder. (the bytecoin folder in AppData).

Still my balance appears as 0.00.

I have a maths/computing degree and have worked in software dev for years, so I think it's very unlikely to be a usability related issue.

This coupled with the fact that it's impossible to get hold of anyone with any bytecoin technical know-how I think confirms that bytecoin is a scam.

**Comment by pmitchev** [comment]
Is your wallet synced? You can download the blockchain from here:
https://bytecoin.org/downloads/

I use a slightly modified CLI and it works quite well, if we exclude some bugs.

**Comment by xranby** [comment]
you need to contact the support of "changer.com"
did they really send the coins?

**Comment by Ben19904** [comment]
@pmitchev please re read my original post

@xranby changer.com have good reputation (and it's very easy to contact them for help/support), and I can see that the transaction is complete from chainradar.

How do I check if the "transaction is still part of the main chain"? Also how do I know there was a chain split, whatever that is?

**Comment by xranby** [comment]
Technical documentation: https://bytecoin.org/documentation/

The blockchain is documented in:
https://cryptonote.org/cns/cns003.txt

first check that the transaction that you can see on chainradar also can be found in your clients blockexplorer. your client need to see that transaction!

try use the clients block explorer and compare if your clients top block's number and hash is the same as other public external block explorers such as chainradar, minergate and democats

if your top block differ then your client have not yet reached consensus what is the main branch of the blockchain

**Comment by nibbons** [comment]
Either you messed something up or Changer did.  If you gave them the correct address, then your coins would be in your wallet.  Are you sure your wallet contains the same address that you used at Changer?  I've used bytecoin for years and it is never "unstable" and coins never go missing...numerous transactions back and forth over the years and many recently...

**Comment by nibbons** [comment]
Does your wallet.cfg file point to the right wallet?

**Comment by nibbons** [comment]
Also try: File > Reset Wallet (it won't delete your wallet it will just rescan the block chain--it wont redownload, it will just rescan all transactions).

**Comment by Teja-ishwar** [comment]
@Ben19904
I know exactly what you are going through. Inam having exactly the same problem. 
I bought Litecoin to Bytecoin from Changer.com
At changer they told me there was lots of issues with Bytecoin and the resync was the problem. 
At first changer.com replied to my email or chat. Now they are just ignoring me. As for Bytecoin they ve never replied to my emails even though they claim on their site that they will answer in 24 hours. I agree with you, Bytecoin is a complete scam. That's cost me 1,500€......
From what I am reading around, we are not the only onea in this case and all these amounts start building up. 
As for the users that tell you that there are no problems with Bytecoin,  can they explain me why so many of us are having problems only with Bytecoin sync and no support from them. Coincidence?

Anybody with suggestions on how to go about this is welcome.

**Comment by andre-amorim** [comment]
Regards to "Bytecoin and the resync was the problem"
does anyone else is getting msg

"2017-May-31 09:54:54.094964 WARNING [xxx.xxx.xxx.xxx:46478 INC] Exception
in connectionHandler: Levin signature mismatch"

0x03

**Comment by Ben19904** [comment]
@nibbons 

I never have more than 8 peers.

The moment after downloading Bytecoin Wallet 1.1.1, I opened it for the first time and I copied the address (using the in-built functionality that auto copies it to your clipboard). This was before any syncing took place. I then pasted this address into the changer.com trade dialog. I then cross-checked the address before confirming the trade.

Now here's a key event that worried me; shortly after confirming the trade, the address in my Bytecoin Wallet 1.1.1 changed to something different. Bear in mind it had barely even begun synchronising at this point.

After this (because the blockchain download link on bytecoin.org was broken at the time) it took a further 7 days to synchronise the block chain to the block that my transaction was a part of. Chainradar suggested my transaction was complete at this time, but still my BCN balance was 0.00.

Until this point I did absolutely no meddling with the bytecoin files (i.e. in the relevant roaming AppData folder on windows), in fact I didn't even know where these files existed at this point.

After realising something was wrong I backed up the .wallet file before continuing onto downloading the new Bytecoin Wallet 1.1.4. After this I realised the bytecoin.org blockchain download link was working again, so I downloaded that as well. I then deleted all files (including DB folder) in the respective app data folder except the bytecoinwallet.cfg, and bytecoinwallet.wallet files. I then unzipped and then copied and pasted the blockchain files into the app data folder, before starting up the new Bytecoin Wallet 1.1.4 for the first time. It took 1+ hours to start up, as it was importing the blockchain; as it was doing this I noticed the DB folder was recreated in the app data folder and it was gradually filling up.

After the importing finished, there were still 1-2 thousand extra blocks that the new wallet still had to sync. This took another few hours. After syncing finished, I can't remember what the block height was, but I know it was much higher than the block in which my transaction was included (I think it was ~1,276,000, and my transaction was in a block less than 1,270,000).

My balance was still 0.00. I then tried opening the backup wallet (although not much point as I never deleted the original .wallet file), and still was 0.00.

Although I have no expertise in networking or cryptography, after working with Linux as a developer, I'm all too familiar with the unforgiving mistakes of certain technologies, and so whilst I was moving/modifying/deleting files in regards to the bytecoin wallet, I was in the same 'careful mode' that I would be in as if I was working with a Linux server; like I said, I am no IT fool, and so I am asserting my issues are not down to any kind of foolish mistake.

**Comment by Teja-ishwar** [comment]
@Ben19904 

http://bytecoindb.blogspot.com.co/2017/04/hi-everyone-this-is-exclusive-blog-to.html?m=1#comment-form

Have you had a look at this link. There s sone help with a folder to download and install. Apparently you are advanced in IT so I thought you have more much than me at trying it and letring me know if this helps/work. 
I'm in exactly same spot as you. It's been a week today. 
My concern is that my Bytecoin software is still synching... 301546/1277930.
Do you have this too? Do you know what happens when synch is over? I guess I'll have to wait till the end to figure out by myself.

**Comment by boomworking** [comment]
let's use online walletfreewallet.org on mobile
if you not android just use Andy to simulate for temporary usage.
until they release iOS version﻿

**Comment by dhimport** [comment]
May I ask you why the coins I sent from wallet A to wallet B do not show up on wallet B ? I made 2 transactions and both are confirmed, yet not showing up on wallet B... scary :)

**Comment by Ben19904** [comment]
changer.com (the exchange I used for my bytecoin transaction) have informed me that they have disabled the buying of bytecoin for now as they keep getting complaints; see here on this link: https://www.changer.com/ , the link for bytecoin in the "receive" column (upon clicking "show more") is now greyed out.

There is something seriously wrong with either the bytecoin network or the bytecoin wallet, and it needs to be looked into.

Does anyone with some "rep" around the bytecoin community know how we can go about contacting a developer directly?

I'm a bit worried... these developers seem to live in the shadows...

**Comment by Sam-GI** [comment]
@Ben19904 Is this resolved yet?

**Comment by Teja-ishwar** [comment]
Yes. Thank you for asking. I replied on another thread. Basically Changer.com returned me the coins. Luckily it never reached the Bytecoin wallet. Bytecoin developers never ever replied to my emails. I'm quite gutted by that. Many people lost their money because of them.

**Comment by Sam-GI** [comment]
Where can I find those people?

**Comment by ccarnivore** [comment]
hey guys, is there another current problem in the blockchain?
can't get my daemon to fully sync. always hangs at block height 1300027.
I've reset the block db twice and imported the db from download file (but its only till end of may)

what to do now?

**Comment by ciscohite** [comment]
You can use your wallet as light wallet and use it without downloading the complete blockchain. Just go to preference --> Remote Daemon and change the value to -

bcnrps.ebrahma.com port 8081

you may check this link for more details - http://knowbytecoin.blogspot.in/2017/08/how-to-get-blazing-fast-blockchain-sync.html

**Comment by jplakali** [comment]
i transferred some bytecoin from cryptonator exchange to bytecoin desktop wallet and trying to sync the wallet but its syncing at very slow speed. i changed the node to 8080 and my wallet synced in 5 minutes but my wallet balance was still 0 so i changed the node to automatic and it started syncing again from 300k blocks. is there any way to take BCN out of the wallet without syncing the wallet? i can see confirmed transaction on chainradar. please help me.

**Comment by pars0097** [comment]
bad support
i sent several messages to support team and i got 0.0 reply :/

**Comment by b0n0** [comment]
get inside the bytecoin.uk discord chat and ask for help there. i never heard of support team for bytecoin. github is for development.
sync your wallet faster from https://blockchain.bytecoin.org/
gl
